### ISSUE TITLE
Add documentation for using docker on LSfR

### ISSUE THREAD

**Issue by sjspielman** [AlexsLemonade/OpenScPCA-analysis#539]
We should add a page with instructions for how to use docker containers on LSfR. We know that Docker is installed already on LSfR, but we need to investigate its usage further (e.g., is it a GUI or only CLI?)

**Comment by jashapiro** [comment]
We could probably remove the `sudo` requirement by adding a `docker` group during setup scripts: https://docs.docker.com/engine/install/linux-postinstall/

**Comment by sjspielman** [comment]
> We could probably remove the sudo requirement by adding a docker group during setup scripts

@jashapiro Just noticed this comment.. do we think this needs more discussion, or is it something we'll likely go ahead and do? I'll write the docs accordingly.

**Comment by jashapiro** [comment]
I filed an issue for it: https://github.com/AlexsLemonade/OpenScPCA-infra/issues/40 but that is as far as I got. I would expect it to be done, I think.

**Comment by sjspielman** [comment]
> I filed an issue for it: https://github.com/AlexsLemonade/OpenScPCA-infra/issues/40 but that is as far as I go

Ahhh, I'm not added to that repo so didn't see it go through! permissions :games:

**Comment by jashapiro** [comment]
Done now: try launching a new instance and you should be able to run docker commands without sudo.

**Comment by sjspielman** [comment]
> Done now: try launching a new instance and you should be able to run docker commands without sudo.

@jashapiro we may have lost docker? This is from a new VSCodium virtual computer I just created.

```
lightsail-user@ip-172-26-0-56:~$ docker pull public.ecr.aws/openscpca/simulate-sce:latest
Command 'docker' not found, but can be installed with:
snap install docker         # version 24.0.5, or
apt  install docker.io      # version 24.0.5-0ubuntu1~22.04.1
apt  install podman-docker  # version 3.4.4+ds1-1ubuntu1.22.04.2
See 'snap info docker' for additional versions.
lightsail-user@ip-172-26-0-56:~$ which docker
lightsail-user@ip-172-26-0-56:~$ 
```

**Comment by sjspielman** [comment]
Yup, works! Just needed a little more time for installs! I was able to successfully run commands in https://github.com/AlexsLemonade/OpenScPCA-analysis/issues/539#issuecomment-2200711299 without needing `sudo`.

**Comment by sjspielman** [comment]
Closed by #557
